scripts/python/generate_langgraph_diagram.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from dotenv import load_dotenv
+
+from agents.graph import KymaGraph
+
+load_dotenv()
```

Comment:
Can you write comments on how to run this script and what env variables are needed to run this script?